openapiGenerator.go

```diff
@@ -416,16 +422,21 @@ func (g *openapiGenerator) generateMessageSchema(message *protomodel.MessageDesc
 	}
 	o := openapi3.NewObjectSchema()
 	o.Description = g.generateDescription(message)
-	msgRules := g.validationRules(message)
-	g.markerRegistry.MustApplyRulesToSchema(msgRules, o, markers.TargetType)
+	if !g.disableValidation {
+		msgRules := g.validationRules(message)
```

Comment:
could you add this check inside `validationRules()` and return an empty list so that this condition is handled directly in the validation rule parser itself?